src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -79,14 +97,53 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.relaxed());
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```

Comment:
So you *may* need to change that whitelist back to "Whitelist.relaxed()" because otherwise it will just clean all of the html out, and then our nice richText editor doesn't actually make rich text. I'm surprised that this works for the images, as it looks like you're matching to html? But maybe I'm misunderstanding something. Also, can you maybe change this variable back to text? I'm not sure if "userText" adds any more information (though you are welcome to argue with me, I could certainly be wrong), and it might break some code later on in this function, unless you are sure you replaced all instances of "text". 

Also, if you need the html Whitelist to be none, we might need two separate variables - "text" (which I think I used later for rich text formatting) and "textWithHTMLRemoved" (Or something nicer) rather than userText

Reply:
It looks like we have "cleanText" down below all of your changes - what if you move that up to here, and then use cleanText in all of your changes? It looks like it should be the exact same thing as your "userText"

Reply:
I have no idea why we need to use the Whitelist.relaxed() but I changed it shouldn´t be a problem but with the changes at the page I can check if still works I guess it does but only in the AppEngine, and now the float of sentiment uses the text thing

Reply:
So whitelist.relaxed() keeps safe html, whitelist.none() gets rid of all html. So if you were to put html to make something bold, it would not work with whitelist.none(). Jsoup's purpose is to clean the content so that you can't put scary scripts and stuff in your messages. If that makes any sense?

Reply:
It does thanks